package.json

```diff
@@ -1,7 +1,7 @@
 {
   "license": "ISC",
   "scripts": {
-    "start": "run-s utils:hash start:sign start:ui",
+    "start": "run-s utils:hash utils:build start:sign start:ui",
```

Comment:
`utils:build` is redundant since _serve_ already builds the files.

Reply:
@pavlovcik

Reply:
Also my fault! I guess I should've asked you to review my pull request to devpanther's fork 😂